src/bootstrap/native.rs

```diff
@@ -179,7 +179,22 @@ fn download_ci_llvm(builder: &Builder<'_>, llvm_sha: &str) {
     let filename = format!("rust-dev-nightly-{}.tar.xz", builder.build.build.triple);
     let tarball = rustc_cache.join(&filename);
     if !tarball.exists() {
-        builder.download_component(base, &format!("{}/{}", url, filename), &tarball);
+        if !tarball.exists() {
```

Comment:
This check looks duplicated - no need to check it twice.

Reply:
oops - that was an accident, thanks for the catch!